pkg/test/usersignup/usersignup.go

```diff
@@ -26,20 +26,22 @@ func WithOriginalSub(originalSub string) Modifier {
 	}
 }
 
-func Approved() Modifier {
+// ApprovedManually sets the UserSignup states to [`approved`]
+func ApprovedManually() Modifier {
 	return func(userSignup *toolchainv1alpha1.UserSignup) {
-		states.SetApproved(userSignup, true)
+		states.SetApprovedManually(userSignup, true)
 	}
 }
 
-func ApprovedAutomatically(before time.Duration) Modifier {
+// ApprovedManuallyAgo sets the UserSignup state to `approved` and adds a status condition
+func ApprovedManuallyAgo(before time.Duration) Modifier {
```

Comment:
Maybe I'm missing something, but what does "ago" in the function name imply? It gave me an impression that it is adding a condition to already approved usersignups, but that isn't confirmed anywhere.

Reply:
I guess it's because it accepts the `before` parameter and that will be reflected in the condition `LastTransitionTime`. Another option for the func name would be something like `ApprovedManuallyEarlier` but not sure it's much better.

Reply:
yes, the `Ago` suffix in this func's name refers to the `duration` arg, which is used to set the `LastTransitionTime` at `now - duration`, ie, somewhere in the past.